packages/fxa-react/images/close.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill="rgba(12, 12, 13, .8)" d="M9.414 8l5.293-5.293a1 1 0 00-1.414-1.414L8 6.586 2.707 1.293a1 1 0 00-1.414 1.414L6.586 8l-5.293 5.293a1 1 0 101.414 1.414L8 9.414l5.293 5.293a1 1 0 001.414-1.414z"/></svg>
```

Comment:
This is needed to allow CSS to style the SVG (set the class to `fill-current` so the fill is inherited from the text color)

Reply:
I thought this only worked in chrome, but I hadn't checked in a couple years, very cool we support it now :)